public/manager-styles.css

```diff
@@ -1,105 +1,107 @@
 /* Cover sliver that goes along top and to the right */
 #root > div {
-    background-color: #404049;
```

Comment:
Can you leave the white space as is? We can clean up formatting in a separate PR if needed.